testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -10747,8 +10747,30 @@ void assertSubFunReturns(boolean binary, String s, int start,
           "VARCHAR(5) NOT NULL");
     };
     final List<SqlLibrary> libraries =
-        list(SqlLibrary.BIG_QUERY, SqlLibrary.ORACLE, SqlLibrary.REDSHIFT,
-            SqlLibrary.SPARK);
+        list(SqlLibrary.BIG_QUERY, SqlLibrary.ORACLE, SqlLibrary.SPARK);
+    f0.forEachLibrary(libraries, consumer);
+  }
+
+  @Test void testGreatestPgFunc() {
```

Comment:
Add jira link

Reply:
Added Javadoc comments for the Jira ticket.